README.md

```diff
@@ -1,13 +1,29 @@
+# YaST - rewrite of the storage library
 
-This is a technoligy preview to demonstrate:
+[![Build Status](https://travis-ci.org/yast/yast-storage-ng.svg?branch=master)](https://travis-ci.org/yast/yast-storage-ng)
+[![Coverage
+Status](https://coveralls.io/repos/github/yast/yast-storage-ng/badge.svg?branch=master)](https://coveralls.io/github/yast/yast-storage-ng?branch=master)
+[![Code
+Climate](https://codeclimate.com/github/yast/yast-storage-ng/badges/gpa.svg)](https://codeclimate.com/github/yast/yast-storage-ng)
 
-- libstorage-ng supports:
+yast2-storage-ng is a reimplementation of the YaST storage library (yast2-storage)
```

Comment:
NP: `storage library` is not accurate as there is also frontend (clients), probably `storage module` it a bit better.